### ISSUE TITLE
Is it really 2-layer NN? [intro-neural-networks/student-admissions/StudentAdmissions]

### ISSUE THREAD

**Issue by soswow** [udacity/deep-learning-v2-pytorch#81]
I am a bit confused. Up to this point instructor was preparing us for multi-layer NN and how to propagate an error up etc. And last section even have label "Training the 2-layer Neural Network"
But from what I see it is simple perceptron with no layers, but only input x and output. Please correct me if I am wrong.

**Comment by kuzand** [comment]
That's exactly what I was wondering too. If it is supposed to be 2-layer NN (with 1 hidden layer and 1 output layer), then the code for training is completely wrong. First of all, the number of weights used there is equal to the number of inputs/features, whereas if we have a hidden layer there should be additional weights for the connections between the hidden layer and the output layer. Second, the weights and bias for the inputs-hidden layer connections and the weights and bias for the hidden - output layer connections should be updated independently by different formulas. the should be updated. Third, the bias is missing from the code.

**Comment by vishnu-sagubandi** [comment]
Have you people noticed that error term was given as  (y - output)*sigmoid_prime(x)
I think this will come into play when we are using mean squsred error function
Since we are using logloss error function, it should be (y - output)

**Comment by abhiojha8** [comment]
The error-related issues are resolved in PR #250. For other user-specific queries, please reach out to your mentor.
